src/components/Navbar.astro

```diff
@@ -36,7 +36,10 @@ const currentPath = Astro.url.pathname;
 <div class="navbar">
   {
     navbarItems.map((item, index) => (
-      <a class:list={index === 0 ? "navbar-logo" : { active: currentPath === item.link }}  href={currentPath === item.link ? '#' : item.link}>
```

Comment:
apparently pre-commit didn't work for someone before.

Reply:
/shrug